tests/test_pkg_integrity.py

```diff
@@ -1,50 +1,31 @@
 import os
+import shutil
+import mock
 import unittest
 import tempfile
 import pkg_integrity
 
+BASEDIR = os.getcwd()
+TESTDIR = BASEDIR + '/tests/testfiles/pkg_integrity'
```

Comment:
os.path.join() is your friend (though we don't do that as much as we should in autospec).

Reply:
It's been getting better! Not all the way better, but better.

Reply:
It doesn't look like BASEDIR gets used anywhere, is that the case? If so, can you just do 
```TESTDIR = os.path.join(os.getcwd(), 'tests/testfiles/pkg_integrity')```